### PR TITLE
Add TeamViewer Support Collector Latest

### PR DESCRIPTION
**Pull request by maartenpaauw**
After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

### CODE REVIEW

File: Casks/teamviewer-supportcollector.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'teamviewer-supportcollector' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://downloadeu9.teamviewer.com/support/SupportCollector.app.zip'
+  name 'TeamViewer Support Collector'
+  homepage 'https://www.teamviewer.com/en/help'
```
Comment (vitorgalvao) [inline Casks/teamviewer-supportcollector.rb]: I see no mention of this app on this page.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/teamviewer-supportcollector.rb (hunk 1 above)
Comment (m44rt3np44uw) [inline Casks/teamviewer-supportcollector.rb]: There isn't a real page about the Teamviewer Support Collector. All I found was a forum post from a staff member: https://community.teamviewer.com/t5/TeamViewer-11/How-to-submit-a-ticket/td-p/72

Comment (vitorgalvao) [inline Casks/teamviewer-supportcollector.rb]: Then who is this app for? What does it do? Why should it be included in HBC?

Comment (m44rt3np44uw) [inline Casks/teamviewer-supportcollector.rb]: This is an Application you will use when you open a support ticket for Team Viewer. This application will collect all local files (that teamviewer created) on the Mac and zip it to your desktop. Teamviewer support needs this zip file.

**Pull request merged**